features/features.go

```diff
@@ -678,4 +678,12 @@ var (
 						enableIn(configv1.DevPreviewNoUpgrade, configv1.TechPreviewNoUpgrade).
 						enhancementPR("https://github.com/openshift/enhancements/pull/1697").
 						mustRegister()
+
+	FeatureGateNutanixMultiSubnets = newFeatureGate("NutanixMultiSubnets").
+					reportProblemsToJiraComponent("SPLAT").
```

Comment:
should this component be "Cloud Compute / Nutanix Provider" ?

Reply:
changed.